src/unitxt/metrics.py

```diff
@@ -474,6 +474,24 @@ def compute(
         return result
 
 
+class StringContainment(InstanceMetric):
+    reduction_map = {"mean": ["string_containment"]}
+    main_score = "string_containment"
+
+    def compute(
+        self, references: List[Any], prediction: Any, additional_inputs: List[Dict]
+    ) -> dict:
+        references = [r.strip() for r in references[0].split("|")]
```

Comment:
IMO it would be nice if we could avoid this split step in the metric, since a metric already has a way of receiving multiple references. We just need to figure out how to pass multiple references to the metric, from how they are represented in your dataset (where they are in a single string separated by a '|'). I think most suitable would be to have some pre-processing step, that takes your input the single references string, and does the splitting, converting it to a list of strings.
I will try to lookup an example for that.

Reply:
done, we have some extent of control on how the *_robustness datasets are generated, so I refined the field to have multiple references, and adjusted the metric code and the card code